internal/manager/task_clean.go

```diff
@@ -64,7 +65,7 @@ func (j *cleanJob) cleanEmptyGalleries(ctx context.Context) {
 	if err := r.WithTxn(ctx, func(ctx context.Context) error {
 		found := true
 		for found {
-			emptyGalleries, _, err := r.Gallery.Query(ctx, &models.GalleryFilterType{
+			emptyGalleries, err := gallery.Query(ctx, r.Gallery, &models.GalleryFilterType{
```

Comment:
Only related to this method definition change: https://github.com/stashapp/stash/pull/4423/files?diff=split&w=1#r1443579818
No change in behavior expected.